src/tests/playwright/runner.spec.ts

```diff
@@ -18,14 +19,26 @@ test.afterEach(async ({page}) => {
   // @ts-ignore
   // eslint-disable-next-line no-undef
   // noinspection TypeScriptUnresolvedReference
-  const result = await page.evaluate(() => webAudioEvaluate);
+  const result = await page.evaluate(async () => await webAudioEvaluate);
   expect(result).toBeTruthy();
 });
 
+test.describe('Benchmark Tests', () => {
```

Comment:
I am interested in why we have two different groups. (I actually like the idea, but curious about the criteria)

Reply:
ACK